react/components/Box/index.tsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const Box = ({ children, noPadding }) => {
+const propTypes = {
+  /** Content of the box */
+  children: PropTypes.node.isRequired,
+  /** Use the full size of the box. */
+  noPadding: PropTypes.bool,
+}
+
+type Props = PropTypes.InferProps<typeof propTypes>
+
+const Box: React.FC<Props> = ({ children, noPadding }) => {
```

Comment:
`import React,  { FC } from 'react'`, maybe ? 🤔

Reply:
Since the pattern is beeing decided here, just FC looks cleaner... Just my opinion haha 😅

Reply:
Actually... Where is this pattern decided? 😅
Also, I feel that is so much clearer to have interfaces and types being called from `React` but I agree that classes, such as `Component` (but personally I think it should be called `React.Component` as in some tutorials), and functions like hooks should be destructured.

Reply:
Since that we want build a robust code style this discussion must be share with the team because this components will be used as """pattern""" for new components and we know that the devil lives in the details. In my opinion, FC is a better approach because keep the code cleaner.